app/routes/v1/task.py

```diff
@@ -62,15 +62,13 @@ def view_all_tasks(tasks,a):
 def delete_task(current_user_id, current_user_role, task_id):
     if current_user_role == 'employee':
         return jsonify({'error': 'You are not authorized to delete the task'}), 403
-    result = delete(task_id)
+    result = delete(task_id, current_user_id, current_user_role)
```

Comment:
We are checking the role inside the delete function, Then why do we need to check here above too?
`if current_user_role == 'employee':
        return jsonify({'error': 'You are not authorized to delete the task'}), 403`